src/control/pb.rs

```diff
@@ -193,10 +193,10 @@ impl ctx::transport::Server {
     }
 
     fn direction(&self) -> tap::tap_event::ProxyDirection {
-        if self.proxy.is_outbound() {
-            tap::tap_event::ProxyDirection::Outbound
-        } else {
-            tap::tap_event::ProxyDirection::Inbound
+        use ctx::Proxy::*;
```

Comment:
Not needed to change. I've moved away from this pattern (glob importing an enum's variants and then matching on them unqualified), since typoing a variant name isn't a compiler error, just a possible lint warning. That's because if the pattern isn't an actual variant, then it matches as if that were an identifier you wanted to use in the match arm. :shrug: